### ISSUE TITLE
[REQUEST] Wordlist with Multiple Types

### ISSUE THREAD

**Issue by Unkn9wn1** [openbullet/openbullet#590]
hi

After #105 
it's good if you add ability to set multiple types for a wordlist
or add ability to add a wordlist multiple times (with different types)

it's needed for users who have all this types (in their `Environment.ini`) :
'USER:PASS' for targets that only accepts U-P
'EMAIL:PASS' for targets that only accepts E-P
`USERNAME:PASSWORD` for targets that accepts both E-P and U-P
(all of them is needed because of data rules and when using mod configs)

thank you

**Comment by openbullet** [comment]
There is no need to set multiple types, you can simply specify that a config can work with both types (hence why you are given both options in the Other Options -> Data part) and if you really need to do what you are asking you can simply make a new WLTYPE with a more general regex that accepts both.

**Comment by Unkn9wn1** [comment]
true, that was not for rules, my fault
it's needed because there is many configs from other mods
or ones who made by guys with different `Environment.ini`
and it's good to just add a UserPass or EmailPass in our `Environment.ini`
then add wordlist as both types, UserPass/UserPass and Credentials or three of them
then using easily without any problem and there is no need to change config before usage

**Comment by openbullet** [comment]
I won't do that. You can edit your own Environment.ini if you need to.

**Comment by Unkn9wn1** [comment]
i get it, adding multiple types make it messy

just please let users add a wordlist twice with different types
i add a file as `NumerPass` then i want to add it as `Credentials` too
but openbullet does not add it, because their path are same (it don't care that types are different)
should users clone that wordlist file to add it same time with different types?

**Comment by openbullet** [comment]
Okay, I can do that

**Comment by MA38kayseri** [comment]
anyone got a wordlist for me? im new
